ui/pages/confirmations/selectors/confirm.ts

```diff
@@ -28,15 +28,9 @@ export function pendingConfirmationsSortedSelector(
     .sort((a1, a2) => a1.time - a2.time);
```

Comment:
Not introduced by this PR, but perhaps worth some refactor here in future.

`getPendingApprovals` returns a fresh array on every call since it uses `Object.values`.

Then we `filter` which also returns a fresh array.

Ideally, we'd use `createSelector` with the underlying `state.metamask.pendingApprovals` then use `Object.values` and `filter` and `sort` inside a result function.